LiteFarmOrg/LiteFarm
# Lf 2889 allow farmers to nominate a custom crop to be added to core crop catalogue

[Jira Ticket LF-2889](https://lite-farm.atlassian.net/browse/LF-2889)
[Parent Jira Ticket LF-2072](https://lite-farm.atlassian.net/browse/LF-2072)

**Description**
This is the last piece for LF-2072. This adds the nomination_id column to the crop table as well as the frontend Checkbox in the add new crop workflow.

**What's Been done**
- Migrated crop table to add new column
- Updated cropController to retrieve generated nomination_id
- Added new function to nominationController for calls that do not come from the /nomination route
- Updated Model and Slice to include nomination_id
- Added Frontend UI
- Added translations

**To test**
1. Go to crop catalogue and start the add new crop workflow
2. Fill in the new crop details and check(or don't check) the box 
3. Complete the form flow through the varietal creation
4. Check the database in crop table, nomination and nomination_status. There should(or should not) be entries for the crop you entered.
